processors/pybossa_indications/processor.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from itertools import groupby
+
+import os
+import time
+import requests
+import pbclient as pbc
+import logging
+logger = logging.getLogger(__name__)
+
+PYBOSSA_URL = os.environ['PYBOSSA_URL']
+PYBOSSA_API_KEY = os.environ['PYBOSSA_API_KEY']
+PROJECT_ID = os.environ['PYBOSSA_PROJECT_INDICATIONS']
+
+
+def process(conf, conn):
+
+    # Get stuff from the DB
+    query = """
+        SELECT documents.id AS meta_id,
+            fda_approvals.id,
+            documents.name,
+            files.url,
+            fda_approvals.type,
+            fda_approvals.notes,
+            fda_approvals.supplement_number
+        FROM documents
+        INNER JOIN fda_approvals ON documents.fda_approval_id = fda_approvals.id
+        INNER JOIN files ON documents.file_id = files.id
+        WHERE documents.url IS NOT NULL
+        AND fda_approvals.type = 'New or Modified Indication'
+        GROUP BY fda_approvals.id,
+                documents.id,
+                files.id
+        ORDER BY fda_approvals.id
+    """
+
+    rows = list(conn['database'].query(query))
+    if rows:
+        _create_tasks(rows)
+    else:
+        logger.info('No new and modified indications found for tasks creation')
+
+
+def _create_tasks(rows):
+    tasks = []
+    for key, group in groupby(rows, lambda x: x['id']):
+        group = list(group)
+        first = group[0]
+        print(str(first['meta_id']))
+        task = {
+            'supplement_number': str(first['supplement_number']),
+            'approval_type': first['type'],
+            'approval_id': str(first['id']),
+            'meta_id': str(first['meta_id']),
+            'notes': first.get('notes', '')
```

Comment:
Well, I thought you had something in mind with them when writing the initial query, so I left them there and forgot to ask you back about their importance. Great that you spotted them.

As for the `meta_id`, I fully agree. I sincerely can't remember when and why I came up with it.

Will update in minutes.